KitchenLib/src/Customs/CustomGameDataObjects/CustomContract.cs

```diff
@@ -0,0 +1,37 @@
+﻿using KitchenData;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace KitchenLib.Customs
+{
+	public abstract class CustomContract : CustomContractLocalisation<Contract>
+	{
+		// Base-Game Variables
+		public virtual RestaurantStatus Status { get; protected set; }
+		public virtual int ExperienceMultiplier { get; protected set; } = 1;
+
+		public override void Convert(GameData gameData, out GameDataObject gameDataObject)
+		{
+			Contract result = ScriptableObject.CreateInstance<Contract>();
+
+			Main.LogDebug($"[CustomRestaurantSetting.Convert] [1.1] Converting Base");
```

Comment:
Should this be `[CustomContract.Convert]`? Many of the other `CustomGameDataObjects` in this PR have the same issue, with the logged name being different.

Reply:
Yep you're right.
I'll change this up shortly.

It's not too major it's just a debug log

Reply:
Thanks. Agreed, not a big deal. Just seemed like it might be a little confusing.